.bmad-core/tasks/review-story.md

```
@@ -2,373 +2,85 @@
-    status: PASS|CONCERNS|FAIL
-    notes: 'Specific findings'
+**Invoke the `/review` skill with the story number:**
 
-recommendations:
-  immediate: # Must fix before production
-    - action: 'Add rate limiting'
-      refs: ['api/auth/login.ts']
-  future: # Can be addressed later
-    - action: 'Consider caching'
-      refs: ['services/data.ts']
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)